platform_plugin_superset/extensions/filters.py

```diff
@@ -101,24 +123,23 @@ def run_filter(
         """
         course = context["course"]
 
-        user = get_current_user()
-        superset_token, dashboard_id = generate_guest_token(user, course)
-
-        if superset_token:
-            context.update(
-                {
-                    "superset_token": superset_token,
-                    "dashboard_id": dashboard_id,
-                    "superset_url": settings.SUPERSET_CONFIG.get("host"),
-                }
-            )
-        else:
-            context.update(
-                {
-                    "error_message": "Superset is not configured properly. Please contact your system administrator.",
-                    "exception": dashboard_id,
-                }
-            )
+        instructor_dashboard_config = getattr(
+            settings, "SUPERSET_INSTRUCTOR_DASHBOARD", {}
+        )
+        dashboard_uuid = instructor_dashboard_config.get("dashboard_uuid")
+
+        extra_filters_format = getattr(settings, "SUPERSET_EXTRA_FILTERS_FORMAT", [])
+
+        default_filters = [
+            "org = '{course.org}'",
+            "course_name = '{course.display_name}'",
```

Comment:
@mariajgrimaldi do you know how to get the course run here?

Reply:
is course type CourseOverview? try with: course.id.run (the run is part of the CourseKey)

Reply:
thanks, it worked